src/main/java/org/gridsuite/study/server/StudyService.java

```diff
@@ -354,27 +345,48 @@ private void deleteFile(@NonNull File file) {
     }
 
     private void createStudyAsync(File caseFile, String originalFilename, String userId, BasicStudyInfos basicStudyInfos) {
-        AtomicReference<Long> startTime = new AtomicReference<>();
-        startTime.set(System.nanoTime());
         try {
-            UUID importReportUuid = UUID.randomUUID();
             UUID caseUuid = importCase(caseFile, originalFilename, basicStudyInfos.getId(), userId);
+
             if (caseUuid != null) {
-                String caseFormat = getCaseFormat(caseUuid, basicStudyInfos.getId(), userId);
-                NetworkInfos networkInfos = persistentStore(caseUuid, basicStudyInfos.getId(), userId, importReportUuid);
-                LoadFlowParameters loadFlowParameters = LoadFlowParameters.load();
-                insertStudy(basicStudyInfos.getId(), userId, networkInfos, caseFormat, caseUuid, false, toEntity(loadFlowParameters), importReportUuid);
+                emitStudyCreate(basicStudyInfos.getId(), caseUuid, userId);
             }
         } catch (Exception e) {
             LOGGER.error(e.toString(), e);
+            deleteStudyIfNotCreationInProgress(basicStudyInfos.getId(), userId);
         } finally {
             deleteFile(caseFile);
-            deleteStudyIfNotCreationInProgress(basicStudyInfos.getId(), userId);
-            LOGGER.trace("Create study '{}' : {} seconds", basicStudyInfos.getId(),
-                    TimeUnit.NANOSECONDS.toSeconds(System.nanoTime() - startTime.get()));
         }
     }
 
+    @Bean
+    @Transactional
```

Comment:
Doesn't do anything

Reply:
If I'm not wrong, it's opening a transaction for the studyEntity creation (line 380)

Reply:
removed